components/custom-radio-button/css/index.scss

```diff
@@ -0,0 +1,35 @@
+/**
+ * @license EUPL-1.2
+ * Copyright (c) 2021 Robbert Broersma
+ */
+
+@import "../../common/focus/mixin";
+@import "./mixin";
+
+.utrecht-custom-radio-button {
+  @include utrecht-custom-radio-button;
+}
+
+.utrecht-custom-radio-button:checked {
```

Comment:
In addition to `:checked`, we usually also offer `. utrecht-custom-radio-button--checked` (and same for other states) so we can render them in Storybook for visual regression tests.